src/Shared/Components/DeploymentConfigDiff/DeploymentConfigDiff.utils.tsx

```diff
@@ -390,6 +465,8 @@ const getConfigMapSecretData = (
 
         return {
             id: `${resourceType === ConfigResourceType.ConfigMap ? EnvResourceType.ConfigMap : EnvResourceType.Secret}-${currentItem?.name || compareItem?.name}`,
+            pathType:
+                resourceType === ConfigResourceType.ConfigMap ? EnvResourceType.ConfigMap : EnvResourceType.Secret,
```

Comment:
Suggestion:

resourceType === ConfigResourceType.ConfigMap ? EnvResourceType.ConfigMap : EnvResourceType.Secret can be commoned out